### DISCUSSION TITLE
Can you help me

### DISCUSSION THREAD

**Discussion by vbzm** [gavanderhoorn/dominh#24] [Q&A]
Hello, I come from China, I want to know how to write joint type data to the position register of Fanuc robot through RPC, I can complete the writing of xyzwpr, but I have not found a method for joint, nor have I got relevant materials. If possible, I hope you can reply me, thank you ~

**Comment by gavanderhoorn** [comment]
It's been a while since I looked into this. To be honest I've mostly used [gavanderhoorn/comet_rpc](https://github.com/gavanderhoorn/comet_rpc).

> I want to know how to write joint type data to the position register of Fanuc robot through RPC

are you referring to the undocumented RPC protocol on port `tcp://:3002`? If you are, I can't really help you with that.
